API/bench_dataframe.py

```diff
@@ -109,3 +109,11 @@ def bench_sort_values(benchmark, dataframe_dtype_int, ncol_sort):
 def bench_nsmallest(benchmark, dataframe_dtype_int, ncol_sort, n):
     by = list(dataframe_dtype_int.columns[:ncol_sort])
     benchmark(dataframe_dtype_int.nsmallest, n, by)
+
+
+@cudf_benchmark(cls="dataframe", dtype="int", nulls=False, cols=6)
+@pytest.mark.parametrize(
+    "expr", ["a+b", "a+b+c+d+e", "a / (sin(a) + cos(b)) * tan(d*e*f)"]
+)
+def bench_eval_func2(benchmark, obj, expr):
```

Comment:
Does the placement of `obj` relative to `expr`, or other fixtures/parameters matter? I imagine so...

Reply:
Yeah I just realized I forgot to handle the name customization correctly right now. Will fix this afternoon.